packages/spanner/system-test/spanner.js

```diff
@@ -511,6 +511,46 @@ var spanner = new Spanner(env);
       });
     });
 
+    it('should insert and delete multiple rows', function(done) {
+      var id = generateName('id');
+      var id2 = generateName('id2');
+
+      var name = generateName('name');
+
+      table.insert([
+        {
+          SingerId: id,
+          Name: name
+        },
+        {
+          SingerId: id2,
+          Name: name
+        }
+      ], function(err) {
+        assert.ifError(err);
+
+        table.deleteRows([id, id2], function(err) {
```

Comment:
Can you add a test case for deleting multiple rows where each row has a composite key and atleast one of the key should be non string.